### DISCUSSION TITLE
GitHub info is cut off when too long

### DISCUSSION THREAD

**Discussion by Andre601** [squidfunk/mkdocs-material#4120] [Q&A] [answered]
I post this here, since I don't quite see it as a bug on Material for MkDocs' end but rather as a small inconvenience.

The issue is, that when you have a too long text - i.e. having a relatively long version - will the displayed GitHub repository have parts of its text cut off.

Example:
![image](https://user-images.githubusercontent.com/11576465/178520020-adcf5b60-f62c-4961-b9ba-36f63d5fa73a.png)

Not sure if there is any easy fix for this issue, but I would be happy if there was and if it could be shared.

**Comment by squidfunk** [comment, marked as answer]
Thanks for reporting. Maybe it's possible to put ellipsis on the tag name, which is causing the issue in most of the cases.

**Reply by squidfunk** [reply]
Fixed in 862dc03f7.
